### PR TITLE
Fixes to FAST website and DesignToken

### PR DESCRIPTION
**Pull request by nicholasrice**
# Pull Request

## 📖 Description
This PR fixes a few styling disparities between vNext site and what is published on https://www.fast.design/.
There are still a few differences between the sites which are intentional:
1. accentFillRecipe changed, so you'll notice buttons (and any UI using the accentFill recipe) are a different color. This is notable in the preview pane, where the button fill color is slightly darker and the text color is white instead of black.
2. The base-font-size and line height changed, so text of some controls will be smaller.
It also fixes a bug in DesignToken where tokens aliased to other tokens that were assigned derived values did not get their CSS custom property updated when the derived value changed.

### General

- [x] I have included a change request file using `$ yarn change`
- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: change/@microsoft-fast-components-a5da2587-dbcf-4d4a-a54d-b006517c53be.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update default value for bodyFont Desgin Token",
```
Comment (janechu) [inline change/@microsoft-fast-components-a5da2587-dbcf-4d4a-a54d-b006517c53be.json:3]:
```suggestion
  "comment": "Update default value for bodyFont Design Token",
```

**Review by EisenbergEffect — APPROVED** [review]

File: change/@microsoft-fast-foundation-23849ffd-d2e9-429c-b922-f1972b6fe773.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fixes bug in DesignToken where aliased tokens didn't update CSS custom properties .",
```
Comment (janechu) [inline change/@microsoft-fast-foundation-23849ffd-d2e9-429c-b922-f1972b6fe773.json:3]:
```suggestion
  "comment": "Fixes bug in DesignToken where aliased tokens didn't update CSS custom properties.",
```

**Review by janechu — APPROVED** [review]

**Comment by EisenbergEffect** [comment]
Just curious, what happens if those design token updates are not queued in the DOM batch?

**Comment by nicholasrice** [comment]
@EisenbergEffect for some properties the change handler gets called before the ref has updated the the preview property with the element reference, so queueUpdate provides opportunity for that

File: change/@microsoft-fast-components-a5da2587-dbcf-4d4a-a54d-b006517c53be.json (hunk 1 above)
Comment (nicholasrice) [inline change/@microsoft-fast-components-a5da2587-dbcf-4d4a-a54d-b006517c53be.json:3]:
```suggestion
  "comment": "Update default value for bodyFont Design Token",
```

File: change/@microsoft-fast-foundation-23849ffd-d2e9-429c-b922-f1972b6fe773.json (hunk 2 above)
Comment (nicholasrice) [inline change/@microsoft-fast-foundation-23849ffd-d2e9-429c-b922-f1972b6fe773.json:3]:
```suggestion
  "comment": "Fixes bug in DesignToken where aliased tokens didn't update CSS custom properties.",
```

**Review by radium-v — APPROVED** [review]

**Pull request merged**